apps/profile/views.py

```diff
@@ -46,6 +47,12 @@ class ProfileUpdate(LoginRequiredMixin, UpdateView):
     template_name_suffix = '_update'
     fields = ['company', 'info', 'visibility_mail', 'visibility_company', 'visibility_info','visibility_first_name','visibility_last_name']
 
+    def update_user(self):
+        user = self.request.user
+        form = forms.UserForm(data=self.request.POST,instance=user)
+        if form.is_valid():
```

Comment:
Warum hier so? Es gibt ein "form_valid" innerhalb dieser Klasse, da ist dan immer "Post" und das Formular Valide.

Reply:
ist die form_valid in der Klasse nicht nur auf profile_image bezogen ?